eng/blockReflectionAttribute.targets

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project>
+  <!--
+    Providing a definition for __BlockReflectionAttribute in an assembly is a signal to the .NET Native toolchain
+    to remove the metadata for all non-public APIs. This both reduces size and disables private reflection on those
+    APIs in libraries that include this. The attribute can also be applied to individual public APIs to similarly block them.
+  -->
+  <PropertyGroup>
+    <BlockReflectionAttribute Condition="'$(BlockReflectionAttribute)' == '' and '$(UWPCompatible)' == 'true' and '$(IsTestProject)' != 'true'">true</BlockReflectionAttribute>
+    <BlockReflectionAttribute Condition="'$(MSBuildProjectExtension)' != '.csproj'">false</BlockReflectionAttribute>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(BlockReflectionAttribute)'=='true'">
+    <CoreCompileDependsOn>$(CoreCompileDependsOn);AddBlockReflectionAttribute</CoreCompileDependsOn>
+    <BlockReflectionAtributeFile>$(MSBuildThisFileDirectory)/BlockReflectionAttribute.cs</BlockReflectionAtributeFile>
+  </PropertyGroup>
+  <Target Name="AddBlockReflectionAttribute">
+    <ItemGroup>
+      <Compile Include="$(BlockReflectionAtributeFile)" />
```

Comment:
typo: attribute

Particularly insidious in a non compiled language 😨

Reply:
its a consistent misspelling so even if it were compiled it'd still work 😛 https://github.com/dotnet/buildtools/blob/28606b8791ccf507651febbf41c480dd87d4a507/src/Microsoft.DotNet.Build.Tasks/PackageFiles/Build.Common.targets#L314